src/index.html

```diff
@@ -20,34 +22,34 @@
 
       <table class="game-field">
         <tbody>
-          <tr class="field-row">
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
+          <tr class='field-row'>
```

Comment:
Replace single quotes with double quotes for consistency in class attribute values. Just discard all changes in this file